src/components/subscriptions/SubscriptionManagementPage.jsx

```diff
@@ -54,9 +56,9 @@ export default function SubscriptionManagementPage() {
               <SubscriptionDetails />
             </div>
             <div className="col-12 col-lg-4 text-md-right">
-              <button className="btn btn-outline-primary">
-                Request assistance
-              </button>
+              <a href={`mailto: ${CUSTOMER_SUPPORT_EMAIL}`} className="btn btn-outline-primary">
```

Comment:
In react we use `link` at the place of `a` and `to` at the place of `href`. So this would be
<Link to={`mailto: ${CUSTOMER_SUPPORT_EMAIL}`} className="btn btn-outline-primary">
Contact Customer Support
</Link>

Reply:
Thanks for the suggestion, Link is not the best solution though. I found `MailtoLink` from `@edx/paragon` that better suits this scenario. I am using that.